stylesheet.css

```diff
@@ -1,3 +1,8 @@
 #ddterm-panel-icon > StIcon {
     -st-icon-style: symbolic;
 }
+
+.indicator-icon {
```

Comment:
like this, and test ok:

https://github.com/oae/gnome-shell-pano/blob/d0d71f8015ada661f35a1390502c8e09fa2a10b6/src/styles/stylesheet.css#L189-L192

Reply:
This will only affect ddterm and will not affect other plugins

Reply:
No, this will affect any UI element with `indicator-icon` style.

For example, if you add this to the stylesheet:

```css
.system-status-icon {
    icon-size: 8px;
}
```

all icons will become small, not only ddterm's one